docs/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+tocMaxDepth: 2
+---
+
+import { WideTile, Flex } from '@redocly/developer-portal/ui';
+
+# Redocly CLI
+
+Redocly CLI is an open source command-line tool for working with OpenAPI descriptions, developer portals, and other API lifecycle operations including API linting, enhancement, and bundling.
+
+<div>
+  <Flex justifyContent="space-between" flexWrap="wrap">
+    <WideTile to="../redoc/deployment/cli.md" header="Redocly and Redoc">
+      Redoc is our open source API documentation tool. Use `redocly` (this project) to work with your Redoc documentation projects.
+    </WideTile>
+    <WideTile to="./file-management.md" header="Manage OpenAPI descriptions">
+      Split an OpenAPI description into logical chunks, bundle the chunks to create a single file, or even join existing definitions into one.
+    </WideTile>
+    <WideTile to="./decorators.md" header="Transform OpenAPI descriptions">
+      Don't have access to the source code? Use decorators to add, change, or remove content from generated OpenAPI descriptions.
+    </WideTile>
+    <WideTile to="./api-standards.md" header="API governance, batteries included">
+      With our ready-made rulesets, check that your API meets a common standard every time it changes. Make use of the built-in and configurable rules to compose the API standards that fit each of your APIs.
+    </WideTile>
+    <WideTile to="https://github.com/Redocly/redocly-cli/blob/main/.github/CONTRIBUTING.md" header="Want to contribute?">
+      Then join our active and supportive community!
+    </WideTile>
+  </Flex>
+</div>
```

Comment:
Let's keep the bit about Redocly and Redoc from the existing index page, I think it's a common question